ADBench/plot_graphs.py

```diff
@@ -172,8 +162,40 @@ def div_lists(alist, blist):
 
 
 
-if do_help:
-    ref_msg = f'''
+def get_style(tool, tool_names):
+    '''Returns the style of the given tool from all tool list.'''
+
+    idx = tool_names.index(tool)
+    
+    # Additional operation for handling the case when tool count is greater
+    # than all style count
+    idx %= len(all_styles)
+
+    return all_styles[idx]
+
+
+
+def  draw_vertical_lines(sorted_vals_by_tools):
+    '''Adds vertical lines to the figure for clarifying results.'''
+
+    n_vals = [ n_val for _, n_val, _, _ in sorted_vals_by_tool ]
+    max_n_val = max(n_vals, key = lambda n_val: len(n_val))
+
+    for n in max_n_val:
+        pyplot.axvline(n, ls = '--', color = "grey", zorder = 0.0, lw = 0.5)
```

Comment:
Can we use non dashes and light grey please.

Reply:
Of course, I will change it as soon as possible

Reply:
Changed. Is this better?

Reply:
Great thanks.